packages/svelte2tsx/test/helpers.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const assert = require("assert");
+function benchmark(fn) {
+	return -Date.now() + (fn(), Date.now());
+}
+function readFileSync(path) {
+	return fs.existsSync(path) ? fs.readFileSync(path, "utf-8").replace(/\r\n/g, "\n").replace(/\s+$/, "") : null;
+}
+function check_dir(path, { allow, require = allow }) {
+	if (allow !== require) allow = [...require, ...allow];
+	const unchecked = new Set(require);
+	const unknown = new Set();
+	a: for (const fileName of fs.readdirSync(path)) {
+		for (const name of unchecked)
+			if (name === fileName || (name.startsWith("*") && fileName.endsWith(name.slice(1)))) {
```

Comment:
Why the `startsWith("*")`?

Reply:
dirty way to support checking for a file format
an input with `'*.svelte'` would check for files that end with `.svelte` instead of trying an exact match